static/app/components/replays/unmaskAlert.tsx

```diff
@@ -1,40 +1,52 @@
 import styled from '@emotion/styled';
 
 import {Button} from 'sentry/components/button';
+import ExternalLink from 'sentry/components/links/externalLink';
+import useUserViewedReplays from 'sentry/components/replays/useUserViewedReplays';
 import {IconClose, IconInfo} from 'sentry/icons';
-import {t} from 'sentry/locale';
+import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import useDismissAlert from 'sentry/utils/useDismissAlert';
 
-const LOCAL_STORAGE_KEY = 'replay-player-dom-alert-dismissed';
+const LOCAL_STORAGE_KEY = 'replay-unmask-alert-dismissed';
 
-function PlayerDOMAlert() {
+function unmaskAlert() {
```

Comment:
feels like this should be uppercase because it's a component.

Reply:
oh, good catch